### ISSUE TITLE
Inheritance mapping with customized mapper - custom mapper not fired for parent class

### ISSUE THREAD

**Issue by GoogleCodeExporter** [vadbch/orika#7]
```
What steps will reproduce the problem?
1. Run the attached JUnit test

What is the expected output? What do you see instead?
The object should map flawlessly, yet the customized mapper for parent class is 
not fired.

What version of the product are you using? On what operating system?
orika-core 1.0, JDK 1.5

Please provide any additional information below.


```

Original issue reported on code.google.com by `lighteater` on 8 Mar 2012 at 11:01

Attachments:
- [OrikaInheritanceTest.java](https://storage.googleapis.com/google-code-attachments/orika/issue-7/comment-0/OrikaInheritanceTest.java)

**Comment by GoogleCodeExporter** [comment]
```
Attaching the same customized mapper to each subclass classmap does the trick, 
but it's an inconvenient workaround :) 
```

Original comment by `lighteater` on 8 Mar 2012 at 11:05

**Comment by GoogleCodeExporter** [comment]
```
There's a fix for this case in the new 1.1-generics branch. It should make it 
into the next release.
```

Original comment by `matt.deb...@gmail.com` on 9 Mar 2012 at 8:16

**Comment by GoogleCodeExporter** [comment]
```
Thanks
```

Original comment by `lighteater` on 9 Mar 2012 at 8:30

**Comment by GoogleCodeExporter** [comment]
```
[deleted comment]
```

**Comment by GoogleCodeExporter** [comment]
```
(was fixed in 1.1.0 release)
```

Original comment by `matt.deb...@gmail.com` on 4 Aug 2012 at 5:06
